packages/account/src/Sections/Profile/PersonalDetails/personal-details-form.tsx

```diff
@@ -425,7 +425,9 @@ const PersonalDetailsForm = observer(() => {
                                                 is_phone_number_editted: account_settings.phone !== values.phone,
                                             })}
                                             onChange={(e: ChangeEvent<HTMLInputElement>) => {
-                                                handleChange(e);
+                                                let phone_number = e.target.value.replace(/[^0-9]/g, '');
```

Comment:
this regex is replacing `+` with empty string as well

Reply:
Nope + will still remain the the input field, i just wanted user to enter numbers only + will be fixed in the inputfield 🙏